deploy/kubernetes/elastic-agent-managed/elastic-agent-managed-daemonset.yaml

```diff
@@ -143,3 +160,10 @@ spec:
           hostPath:
             path: /var/lib/elastic-agent-managed/kube-system/state
             type: DirectoryOrCreate
+        # Mount required for Universal Profiling.
+        # If you are not using this integration, then these volumes and the corresponding
+        # mounts can be removed.
+        - name: universal-profiling-cache
```

Comment:
Can we have those initially commented out?

Reply:
sure - I will comment this out.

Reply:
done :white_check_mark: